src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/NotifichePFPage.java

```diff
@@ -127,12 +129,9 @@ public boolean getListData() {
     public boolean verificaEsistenzaEPassaggioPagina() {
         this.js().executeScript("window.scrollBy(0,document.body.scrollHeight)");
         try {
-            By numeroButtonBy = By.id("page2");
-            this.getWebDriverWait(20).withMessage("bottone pagina 2 non è visibile").until(ExpectedConditions.visibilityOfElementLocated(numeroButtonBy));
+            this.getWebDriverWait(20).withMessage("bottone pagina 2 non è visibile").until(ExpectedConditions.visibilityOf(paginaSeconda));
```

Comment:
By default we usually put this **10 seconds** because it shouldn't take so much time to load a button if so there is a problem behind the scene